src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -178,6 +178,62 @@ end
 
 
 
+######################
+# Equality
+######################
+
+@doc raw"""
+    Base.:(==)(tv1::NormalToricVariety, tv2::NormalToricVariety)
+
+In OSCAR, the `==` operator is reserved to check if two normal toric varieties are identical,
+meaning their underlying polyhedral fans are the same. However, this check is computationally
+challenging due to several reasons:
+
+- The ray generators might be scaled.
+- The ray generators might be stored in different orders.
+- The maximal (polyhedral) cones of the polyhedral fan might be stored in different orders.
+- If we fall back on polyhedral fan equality, lineality of the cones must also be considered.
+
+To avoid this computational bottleneck, we have designed OSCAR to *not* implement the `==`
+method for toric varieties. Instead, using `==` will always raise an error, as shown below.
+
+However, triple equality `===` is supported and will return `true` if the two instances of
+normal toric varieties refer to the same information on the hard drive (as sequence of bits and
```

Comment:
This is technically not correct. === checks if the memory locations are identical. This can return false, even if both objects have an identical memory representation, just being stored at different places in memory.

Reply:
Thank you.

Reply:
I have just fixed this locally and will push the corresponding change in a moment.